src/esp/gfx/replay/Recorder.cpp

```diff
@@ -162,9 +162,13 @@ void Recorder::advanceKeyframe() {
   currKeyframe_ = Keyframe{};
 }
 
-void Recorder::writeSavedKeyframesToFile(const std::string& filepath) {
+void Recorder::writeSavedKeyframesToFile(const std::string& filepath,
+                                         bool usePrettyWriter) {
   auto document = writeKeyframesToJsonDocument();
-  esp::io::writeJsonToFile(document, filepath);
+  // replay::Keyframes use floats (not doubles) so this is plenty of precision
+  const float maxDecimalPlaces = 7;
```

Comment:
Maybe make an argument with a default value? (standard magic number stuff, not a big deal)

Reply:
I don't want to let the user make a bad decision here like setting it to 0 or 2. This is a decision for Recorder.